### ISSUE TITLE
Email notifications not sent out after updates to the event

### ISSUE THREAD

**Issue by 19diesel86** [lowercasename/gathio#47]
Hi,

First of all, thanks for the great project!

I've created an event and attendees confirmed their participation providing an email address, so as to receive updates. While they did receive an email confirming that their participation was registered in the system, they are not getting any notification about changes to the event.

Thanks!

**Comment by lowercasename** [comment]
Hi! Sorry it's taken a while to get back to you! I've put this on the to-do list, so hopefully I'll get a chance to look at it this month. :)

**Comment by lowercasename** [comment]
It's been a minute but I've just run some tests and I can confirm I'm successfully getting emails to my 'attendee' account about comments and updates to an event. It may have been an issue with the email service at that point? I'm going to close this, but absolutely open it again if you still can't receive emails.

**Comment by alexdibella** [comment]
Hey there, nice project! I am not sure this issue has been solved, or maybe I didn't get how notifications work :)

I created an event on the hosted version at gath.io to which attendees have registered their participation and received RSVP confirmations to the email they provided. However, no notification is sent when the time, location or description of the event are changed, nor when comments have been posted. Any ideas why this happen?

Thanks a lot!
